### ISSUE TITLE
Awesome

### ISSUE THREAD

**Issue by volvox89** [tbaccata/amica#2]
I just wanted to quickly express my gratitude for your great tool. I really enjoy using is.
I would only like to suggest to enable different types of depicting protein profile plots across several samples. It would be great if you could enable to depcit all individaul data points in each sample. Another great way to depcit the profile plots, would be via violin plots. 

For getting a large user base, I think it would be good for this tool to function indepentently on Rstudio etc.  I am a little familiar with non R coding and was suffering/troubleshooting a lot to get it running and feel this may be a high entry barrier for people.
Howver, I think it was worth it ;)
Cheers

**Comment by tbaccata** [comment]
Hello!

Thanks for the feedback! Glad you like it! 

Adding an option for the profile plot to include individual data points/violin plots is definitely a useful feature, we will implement this soon.

Could you describe a little bit more in detail which issues you encountered during the local installation? amica should also run without Rstudio.


Best,
Sebastian

**Comment by volvox89** [comment]
Besides a minor R course during my Bachelor (ages ago), I never used R, so I think it's mainly minor problems, which arise from the almost complete lack of R experience. When I was setting up amica according to the instructions, I encountered following issues:
-Was completely unfamiliar with the R-Github interconnection, but found a good youtube tutorial how it works.
-I was however somehow not able to clone the Github Adress into R studio and ultimately manually placed the amica folder structure (I always received the message that the github adress, which I copy/pasted does not exist).
-When installing the dependencies package, I realized that there are some additional dependencies. One (sorry forgot the name) was thankfully suggested by Rstudio itself, but the other one (Rtools) is only indicated in the error messages when installing the dependencies. However there is also a pre-installed tools package in Rstudio, which I first mistook for the Rtools. Took me some additional time to figure out, that it's an additional downloadable package and first wasted some time in getting it to work without Rtools, because I thought it should already be there.
-Several basic nomenclature error messages, which come up when you start using R without knowledge about it
I know it's just minor things, which are probably not an issue if you know what you are doing, but as a newbie, the R envrionment is a little unnerving. I have a little pyhton coding knowledge and I imagine that the installation would have been really frustrating without it. 
I have the impression that one of the big purposes of Amica is the straightforward and easy application of various sophisticated data analysis tools on your dataset without too much knowledge in coding, which is why I think the project would strongly benefit from a standalone executable. I just do not have any idea how feasible/time consuming this would be. 
Best
Andree
